.gitattributes

```diff
@@ -1,9 +1 @@
-* text=auto
-
-*.py text
-*.ipynb text
-*.txt text
-*.md text
-
-*.png binary
-*.jpg binary
```

Comment:
These lines do not affect linguist and can be removed.

Reply:
So, this file is necessary to handle line endings across different Operating Systems.
Read more at: https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings

Reply:
Ah, I didn't know that. Thanks!